src/test/lib/HandshakeTest.cpp

```diff
@@ -2660,31 +2660,40 @@ QuicTestLoadBalancedHandshake(
         TEST_QUIC_SUCCEEDED(Connection.SetLocalAddr(ConnLocalAddr)); // TODO - Put in loop in case addr is taken
```

Comment:
Why not do this instead of just ignoring address in use errors?

Reply:
Actually, I should just remove this comment. The problem is that this won't fail, because we don't actually try to bind the socket until you start the connection, which if bind fails, results in the transport shutdown error code this PR is now handling.